pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
+    height: 70px;
+    border: 1px transparent grey;
```

Comment:
The transparent keyword is unnecessary and incorrect in this context. If you want a transparent border, use border: 1px solid transparent;. If you want a grey border, use a specific shade of grey, like border: 1px solid #ccc;.